CHANGELOG.md

```diff
@@ -13,6 +13,9 @@ When updating the changelog, remember to be very clear about what behavior has c
 and what APIs have changed, if applicable.
 
 ## [Unreleased]
+
+## [29.5.5] - 2020-08-21
```

Comment:
You should also update gradle.properties to the same version before release

Reply:
good catch, so everytime we release, we modify the gradle.properties file?